docs/guides/docs/intro-to-sway/typescript-sdk.mdx

```diff
@@ -0,0 +1,344 @@
+---
+title: Typescript Frontend
+category: Intro to Sway
+parent:
+  label: Guides
+  link: /guides
+---
+
+## Building the Frontend
+
+### Setup
+
+Initialize a new React app with TypeScript in the same parent folder as your contract using the command below.
+
+```shell
+npx create-react-app frontend --template typescript
+```
+
+Next, install the fuels Typescript and wallet SDKs in the frontend folder and generate types from your contract with `fuels typegen`.
+
+```shell
+cd frontend
+npm install fuels@0.60.0 @fuel-wallet/sdk@0.13.0 --save
+npx fuels typegen -i ../contract/out/debug/*-abi.json -o ./src/contracts
+```
+
+Open the `src/App.tsx` file, and replace the boilerplate code with the template below:
+
+<CodeImport
+  file="../../examples/intro-to-sway/frontend/src/App.tsx"
+  comment="fe_app_template"
+  commentType="//"
+  lang="tsx"
+/>
+
+Finally, copy and paste the CSS code below in your `App.css` file to add some simple styling.
+
+<CodeImport
+  file="../../examples/intro-to-sway/frontend/src/App.css"
+  comment="fe_css"
+  commentType="//"
+  lang="tsx"
+/>
+
+### Connecting to the contract
+
+At the top of the file, add your contract ID as a constant.
+
+<CodeImport
+  file="../../examples/intro-to-sway/frontend/src/App.tsx"
+  comment="fe_contract_id"
+  commentType="//"
+  lang="tsx"
+/>
+
+Next, create a new folder in the `src` folder called `hooks`, and copy and paste the `useFuel.tsx` and `useIsConnected.tsx` hooks from the example in this repo. You can also find them in the [offical wallet docs](https://wallet.fuel.network/docs/react/).
```

Comment:
I would change this section, since the hooks can be found at @fuel-wallet/react.

More details [here](https://github.com/FuelLabs/fuels-wallet/blob/docs/fuel-wallet-react/packages/docs/docs/dev/getting-started.mdx#L84C17-L84C17).

Reply:
Yup this is outdated good catch